format/Flight.proto

```diff
@@ -301,6 +301,44 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // This is an optional optimization for applications that want to reduce latency,
+  // for previewing results or retrieving small data sets by providing data
+  // without requiring a DoGet call.  Applications built on top of Flight 
+  // are responsible for any negotiation necessary on whether
+  // inlining data is appropriate.
+  //
+  // The size of inlined_data is expected to be small (typically less then 1MB) 
+  // and inlining too much data across tickets can run into underlying transport 
+  // limitations.  Furthermore, since the data is expected to be small, implementations 
+  // are less likely to optimize for zero-copy in these cases. 
+  message InlinedData { 
+    // Arrow data for consumption.
+    //
+    // The data is not expected to contain the schema message. The schema
+    // should be identical to the schema provided on FlightInfo. 
+    repeated FlightData data = 1;
```

Comment:
Should this be reordered? Such that
inlined_completeness = 1
data = 2
That way if inlined_completeness == DATA_SUBSET_TYPE_UNDEFINED then there would be no need to read data.

This would then follow the similar pattern in FlightDescriptor where DescriptorType precedes the fields cmd and path whose contents or not is defined based on DescriptorType.

Reply:
sure.